src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -61,7 +61,8 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 			this._primaryTarget = localize('executionPlanPropertiesEdgeOperationName', "Edge");
 		}
 
-		let topTitleText = localize('executionPlanComparisonPropertiesTopOperation', "Top operation: {0}", this._primaryTarget);
+		const operationOrientation = this._orientation === ExecutionPlanCompareOrientation.Horizontal ? 'Top operation' : 'Left operation';
```

Comment:
I assume you want `Top operation` and `Left operation` localized, right? You have to create separate localized strings for that to work - with how you have it now you'll end up with those being in english too

Reply:
What I would suggest doing is pulling these out and declaring them at the top of the file (4 of them, one for each "direction") and then the logic here would just choose the one you want to use based on the current orientation.